File: news/src/main/java/material/com/news/ui/PageConfig.java
package material.com.news.ui;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by zjl on 16/7/6.
 */
public class PageConfig {
    private static List<String> pageTitles = new ArrayList<String>();

    public static List<String> getPageTitles() {
        pageTitles.clear();
        pageTitles.add("all");
        pageTitles.add("Android");
        pageTitles.add("iOS");
        pageTitles.add("福利");
        return pageTitles;
    }

    private static final String NewFragment = "material.com.news.ui.NewFragment";

    public static String[] fragmentNames = {
            NewFragment,
            NewFragment,
            NewFragment,
            NewFragment
    };
}

File: news/src/main/java/material/com/news/ui/PageConfig.kt
package material.com.news.ui

import java.util.ArrayList

/**
 * Created by zjl on 16/7/6.
 */
object PageConfig {
    private val pageTitles = ArrayList<String>()

    fun getPageTitles(): List<String> {
        pageTitles.clear()
        pageTitles.add("all")
        pageTitles.add("Android")
        pageTitles.add("iOS")
        pageTitles.add("福利")
        return pageTitles
    }

    private val NewFragment = "material.com.news.ui.NewFragment"

    var fragmentNames = arrayOf(NewFragment, NewFragment, NewFragment, NewFragment)
}
